ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml

```diff
@@ -584,7 +584,7 @@ tests:
     pre:
     - chain: ipi-gcp-pre
```

Comment:
I would fix the job instead of renaming it.  The problem is this job overrides the pre chain, so it's missing the sdn-conf step entirely.

Reply:
Thank you, that is what I was looking for.  Is it enough to just add the    - ref: sdn-conf   ?  Thinking I should also add the explicit ovn-conf too (or just leave it as is?)

   pre:
    - chain: ipi-gcp-pre
    - ref: fips-check
    - ref: sdn-conf

Reply:
/close